SimCalorimetry/EcalEBTrigPrimAlgos/src/EcalEBTrigPrimTestAlgo.cc

```diff
@@ -225,14 +232,14 @@ void EcalEBTrigPrimTestAlgo::run(const edm::EventSetup & setup,
 	  int nSam=0;
 	  for (int iSample=firstSample;iSample<=lastSample;++iSample) {
 	    etInADC= tcpformat_out_[iSample];
-	    if (debug_) std::cout << " format_out " << tcpformat_out_[iSample] <<  " etInADC " << etInADC << std::endl;
+	    //if (debug_) std::cout << " format_out " << tcpformat_out_[iSample] <<  " etInADC " << etInADC << std::endl;
 
-	    bool isASpike=false; // no spikes for now 
-            int timing=0;   //  set to 0  value for now
+	    bool isASpike=0; 
+            int timing=0;
 	    tp.setSample(nSam,  EcalEBTriggerPrimitiveSample(etInADC,isASpike,timing)  );
 
 	    nSam++;
-	    if (debug_) std::cout << "in TestAlgo" <<" tp size "<<tp.size() << std::endl;
+	    //if (debug_) std::cout << "in TestAlgo" <<" tp size "<<tp.size() << std::endl;
```

Comment:
delete commented-out code

Reply:
Please do, or justify why you prefer not doing it